src/encoded/static/components/browse/components/file-tables.js

```diff
@@ -474,7 +504,7 @@ export class RawFilesStackedTable extends React.PureComponent {
 
     renderExperimentBlock(exp,i){
         this.cache.oddExpRow = !this.cache.oddExpRow;
-        const { experimentSet, collapseLongLists, collapseShow = 3, collapseLimit = 2, columnHeaders, showMetricsColumns, href, preventExpand } = this.props;
+        const { experimentSet, collapseLongLists, collapseShow = 3, collapseLimit = 2, columnHeaders, showMetricsColumns, showNotesColumns, href, preventExpand } = this.props;
```

Comment:
Codacy found an issue: ['collapseShow' is missing in props validation](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5860998)